tests/executors/test_dask_executor.py

```diff
@@ -121,27 +120,27 @@ def setUp(self):
 
     @conf_vars(
         {
-            ('dask', 'tls_ca'): get_cert('tls-ca-cert.pem'),
-            ('dask', 'tls_cert'): get_cert('tls-key-cert.pem'),
-            ('dask', 'tls_key'): get_cert('tls-key.pem'),
+            ('dask', 'tls_ca'): 'certs/tls-ca-cert.pem',
+            ('dask', 'tls_cert'): 'certs/tls-key-cert.pem',
+            ('dask', 'tls_key'): 'certs/tls-key.pem',
         }
     )
-    def test_tls(self):
-        # These use test certs that ship with dask/distributed and should not be
-        #  used in production
-        with dask_testing_cluster(
-            worker_kwargs={'security': tls_security(), "protocol": "tls"},
```

Comment:
tls_security in `utils_test.py` part of `distributed` seems to look for the certs in the tests/ path
    `path = os.path.join(certs_dir, filename)`
For now commenting out if its OK, the other tests seem to pass

Reply:
Let's see. They did not for me when I run them previously.

Reply:
Atleast on my local, most of the tests were failing because of errors with pickling the function, the change in [airflow/executors/dask_executor.py], fixed it

       - future = self.client.submit(airflow_run, pure=False, resources=resources)
       + future = self.client.submit(subprocess.check_call, command, pure=False, resources=resources)

Reply:
Cool @subkanthi ! Looks good. I think you can bring back the tests now - in my change (when you rebase) you will see how I dealt with get_cert to skip those

Reply:
I think you cna ucomment your line comments now :).